src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Types/UserPickerFieldQueryObjectType.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL;
+using GraphQL.DataLoader;
+using GraphQL.Types;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Apis.GraphQL;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.Users.GraphQL;
+using OrchardCore.Users.Indexes;
+using OrchardCore.Users.Models;
+using YesSql;
+using YesSql.Services;
+
+namespace OrchardCore.ContentFields.GraphQL
+{
+    public class UserPickerFieldQueryObjectType : ObjectGraphType<UserPickerField>
+    {
+        public UserPickerFieldQueryObjectType()
+        {
+            Name = nameof(UserPickerField);
+
+            Field<ListGraphType<StringGraphType>, IEnumerable<string>>("userIds")
+                .Description("user ids")
```

Comment:
I think all descriptions should be localizable.

Reply:
Done